lib/PHPCfg/Parser.php

```diff
@@ -1010,9 +1014,16 @@ protected function parseExpr_Isset(Expr\Isset_ $expr) {
         );
     }
 
+	protected function parseExpr_List(Expr\List_ $expr) {
+		return new Op\Expr\List_(
+			$this->parseExprList($expr->items),
+			$this->mapAttributes($expr)
+		);
+	}
+
     protected function parseListAssignment(Expr\List_ $expr, Operand $rhs) {
         $attributes = $this->mapAttributes($expr);
-        foreach ($expr->vars as $i => $var) {
+        foreach ($expr->items as $i => $var) {
```

Comment:
`$var` is now either null or an ArrayItem, the variable needs to be taken from that ArrayItem first. This is probably the reason why you ran into the undefined methods.